coalib/io/FileFactory.py

```diff
@@ -64,13 +64,14 @@ class FileFactory:
     >>> os.remove(ff.name)
     """
 
-    def __init__(self, filename):
+    def __init__(self, filename, newline=True):
         """
         :param filename:
             The filepath.
         """
         self._filename = os.path.abspath(filename)
         self._timestamp = os.path.getmtime(self._filename)
+        self._newline = newline
```

Comment:
Since the bear writer using the file factory can't control this parameter via `__init__` (because the core instantiates them), we should maybe add additional function(s) (maybe `get_lines(options)`?) that allow you to get different kinds of strings and lines?

Reply:
This was a part of my cep PR iirc